lib/host_runner.rb

```diff
@@ -35,10 +47,60 @@ def start
       raise LockError.new("Host '#{@host_name}' already locked")
     end
 
+    connect
```

Comment:
When the cleanup is skipped there is no need to call `connect`, right? In this case I would skip this as well. Then it wouldn't even needed to have snapper installed on a system.